### DISCUSSION TITLE
Is there a way to run the scripts locally like we used to be able to?

### DISCUSSION THREAD

**Discussion by dan-hoeger** [aws-samples/eks-cluster-upgrade#57] [Q&A]
we had used this but then modified the AMI's to use our custom AMI's but now that it's launched with eksupgrade it doesn't seem to use the local code with our changes.

I tried launching it with a copy of the older "eks_updater.py" but then we lose the logging functionality and only see error messages.

is there a way to setup the logging so it works correctly in the "eks_updater.py"?

alternatively, if there is a better way to allow upgrade of "custom ami's" 

Thanks,

Dan

**Comment by bryantbiggs** [comment]
Hi @dan-hoeger - this is a great question. This is something we are looking at incorporating since we recognize a number of customers do utilize custom AMIs. If you get some time, I'd love your input/feedback on https://github.com/aws-samples/eks-cluster-upgrade/issues/34 if you wouldn't mind
